forms/ui_channelsPage.py

```diff
@@ -208,16 +208,27 @@ def setupUi(self, ChannelsPage):
         self.line_2.setObjectName("line_2")
         self.layoutNodeId.addWidget(self.line_2)
         self.horizontalLayout_5.addLayout(self.layoutNodeId)
+        self.layoutIncoming = QtWidgets.QVBoxLayout()
+        self.layoutIncoming.setObjectName("layoutIncoming")
+        self.label_10 = QtWidgets.QLabel(self.verticalFrame)
+        self.label_10.setObjectName("label_10")
+        self.layoutIncoming.addWidget(self.label_10)
+        self.line_6 = QtWidgets.QFrame(self.verticalFrame)
+        self.line_6.setFrameShape(QtWidgets.QFrame.HLine)
+        self.line_6.setFrameShadow(QtWidgets.QFrame.Sunken)
+        self.line_6.setObjectName("line_6")
+        self.layoutIncoming.addWidget(self.line_6)
+        self.horizontalLayout_5.addLayout(self.layoutIncoming)
         self.layoutAmount = QtWidgets.QVBoxLayout()
         self.layoutAmount.setObjectName("layoutAmount")
         self.label_8 = QtWidgets.QLabel(self.verticalFrame)
         self.label_8.setObjectName("label_8")
         self.layoutAmount.addWidget(self.label_8)
-        self.line_3 = QtWidgets.QFrame(self.verticalFrame)
-        self.line_3.setFrameShape(QtWidgets.QFrame.HLine)
-        self.line_3.setFrameShadow(QtWidgets.QFrame.Sunken)
-        self.line_3.setObjectName("line_3")
-        self.layoutAmount.addWidget(self.line_3)
+        self.line_ = QtWidgets.QFrame(self.verticalFrame)
+        self.line_.setFrameShape(QtWidgets.QFrame.HLine)
+        self.line_.setFrameShadow(QtWidgets.QFrame.Sunken)
+        self.line_.setObjectName("line_")
+        self.layoutAmount.addWidget(self.line_)
```

Comment:
This is not coherent with the xml file (`s/line_/line_3`), did you modify this file by hand ?

Reply:
apologies, I have re-generated this file and should now be in sync